TypeScript/src/components/Order.ts

```diff
@@ -0,0 +1,120 @@
+type MaybeNumber = number | null;
+
+export class Order{
+    bagsNumber:MaybeNumber;
+    private readonly coffePrice:MaybeNumber;
+    private readonly largeBoxPrice:MaybeNumber;
+    private readonly mediumBoxPrice:MaybeNumber;
+    private readonly smallBoxPrice:MaybeNumber;
+    private readonly large:MaybeNumber;
+    private readonly medium:MaybeNumber;
+    private readonly small:MaybeNumber;
+    private readonly totalPricePercentageDiscount:MaybeNumber;
+    private readonly discountCostRange:MaybeNumber;
+    largeUsed:MaybeNumber;
+    mediumUsed:MaybeNumber;
+    smallUsed:MaybeNumber;
+    rest:MaybeNumber;
+    totalCoffePrice:MaybeNumber;
+    totalLargeBoxesPrice:MaybeNumber;
+    totalMediumBoxesPrice:MaybeNumber;
+    totalSmallBoxesPrice:MaybeNumber;
+    totalPriceInitial:MaybeNumber;
+    totalPriceFinal:MaybeNumber;
+
+    orginizeBags(): void {
+        const number = this.bagsNumber;
+        const result = number % this.large;
+
+        //Large Boxes Part
+        if(result===0){
```

Comment:
For later when we have many ifs and else's let's try putting them in a different file to be cleaner.

Reply:
ok sure